### PR TITLE
README/CONTRIBUTING: Add links to IRC chat

### PR DESCRIPTION
**Pull request by ubruhin**
@dbrgn The IRC<->Telegram bridge is now working properly, isn't it?

So I would say it's time to mention the new chat in README and CONTRIBUTING :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
   [Browse all easy issues](https://github.com/LibrePCB/LibrePCB/labels/easy).
   Some issues even have a mentor assigned:
   [Browse all mentored issues](https://github.com/LibrePCB/LibrePCB/labels/mentored).
-- To contact us use either GitHub or join our [Telegram chat](https://telegram.me/LibrePCB_dev).
+- To contact us use either GitHub or join our [IRC](https://webchat.freenode.net/?channels=#librepcb)
+  or [Telegram](https://telegram.me/LibrePCB_dev) chat.
```
Comment (dbrgn) [inline CONTRIBUTING.md]: Maybe mention that they're bridged?

Comment (ubruhin) [inline CONTRIBUTING.md]: Good point.

> To contact us use either GitHub or join our [IRC](https://webchat.freenode.net/?channels=#librepcb) or [Telegram](https://telegram.me/LibrePCB_dev) chat (they are automatically synchronized).

Better?

Comment (dbrgn) [inline CONTRIBUTING.md]: Yes!

Comment (ubruhin) [inline CONTRIBUTING.md]: fixed.

**Pull request merged**